core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.controller.ts

```diff
@@ -117,23 +141,82 @@ angular.module('oppia').controller('QuestionSuggestionReviewModalController', [
       $scope.validationError = null;
     };
 
+    $scope.refreshModalData = function() {
+      SkillBackendApiService.fetchSkillAsync(
+        $scope.suggestion.change.skill_id).then((skillDict) => {
+        var misconceptionsBySkill = {};
+        var skill = skillDict.skill;
+        misconceptionsBySkill[skill.getId()] = skill.getMisconceptions();
+        $scope.misconceptionsBySkill = misconceptionsBySkill;
+        $scope.init();
+      });
+    };
+
+    $scope.gotoNextItem = function() {
```

Comment:
It seems that you're replicating a lot of the logic here. Can some generic component be created for this and then used in both the translation review and question review?

Reply:
This will be done in a separate PR (1.5)